xknx/io/gateway_scanner.py

```diff
@@ -145,10 +160,15 @@ async def _send_search_requests(self) -> None:
             try:
                 af_inet = netifaces.ifaddresses(interface)[netifaces.AF_INET]
                 ip_addr = af_inet[0]["addr"]
-                await self._search_interface(interface, ip_addr)
             except KeyError:
-                logger.info("Could not connect to an KNX/IP device on %s", interface)
+                logger.debug("No IPv4 address found on %s", interface)
+                continue
+            except ValueError as err:
```

Comment:
I believe this is the error a discord user reported, can we mention this in the changelog just for the sake of completeness?